batch/batch/front_end/templates/table_search.html

```diff
@@ -45,8 +45,8 @@
     }
   </style>
   <div class="flex-col">
-    <div id="{{ component_name }}" class="flex-col-align-right">
-      <form method="GET" action="{{ endpoint }}">
+    <div id="{{ component_name }}" class="flex-col-align-left">
```

Comment:
What does the left vs right do?

Reply:
what do you mean by jumping around? is it moving after the page has finished loading, or do you mean that it's in different places on the batches and jobs pages? if the goal is to get the search box to be aligned to the left side of the page consistently, i tried just changing this to `flex-col` and it did the trick on both the batches and jobs pages.

Reply:
ohh i see what you mean from the screenshots, `flex-col` or `flex-col-align-left` fixes that on my machine